intercom/README.md

```diff
@@ -0,0 +1,84 @@
+# Intercom
+
+## Overview
+[Intercom][1] is a customer communication platform that enables businesses to engage with their users through in-app messaging, email, and chat. It offers features like live chat, automated messaging, and customer support tools, making it easier for companies to provide personalized customer experiences.
+
+The Intercom integration seamlessly collects admin activities, data events, conversations, news items, and ticket data, and ingests them into Datadog for comprehensive analysis using webhooks.
```

Comment:
Can we please add a link to Asana's webhook docs?

Reply:
we have added the link of intercom webhook docs.